.travis.yml

```diff
@@ -8,7 +8,8 @@ env:
   - TRAVIS_NODE_VERSION="4"
 before_install:
   - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
-  - gem install sass
+  - chmod +x ./tool/scissors_dependencies
+  - ./tool/scissors_dependencies
```

Comment:
You probably want `. ./tool/scissors_dependencies` here (or `source ./tool/scissors_dependencies`): this will execute the commands of that script at the top-level, meaning variable exports (like SASSC_BIN) will be visible to the rest of the execution